bool stsRev_CAN # 0 : ko nhân được dữ liệu CAN | 1 : bình thường
float32 voltages # điện áp
uint16 voltages_analog
float32 charge_current # dòng 
uint16 charge_analog
bool stsButton_power # nút on/off
bool status_input1
uint8 temperture # nhiệt độ